### PR TITLE
Update dns-providers.md

### PR DESCRIPTION
**Pull request by futureshield**
added FutureDNS as a provider

### CODE REVIEW

File: docs/general/dns-providers.md [hunk 1]
```diff
@@ -353,6 +353,25 @@ A collaborative open project to promote, implement, and deploy [DNS Privacy](htt
 | DNS-over-TLS | Provider: `Surfnet` Hostname: `tls://dnsovertls.sinodun.com` IP: `145.100.185.15` and IPv6: `2001:610:1:40ba:145:100:185:15`  | [Add to AdGuard](adguard:add_dns_server?address=tls://dnsovertls.sinodun.com&name=dnsovertls.sinodun.com), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=tls://dnsovertls.sinodun.com&name=dnsovertls.sinodun.com) |
 | DNS-over-TLS | Provider: `Surfnet` Hostname: `tls://dnsovertls1.sinodun.com` IP: `145.100.185.16` and IPv6: `2001:610:1:40ba:145:100:185:16`  | [Add to AdGuard](adguard:add_dns_server?address=tls://dnsovertls1.sinodun.com&name=dnsovertls1.sinodun.com), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=tls://dnsovertls1.sinodun.com&name=dnsovertls1.sinodun.com) |
 
+### FutureDNS
+
+[FutureDNS](https://futuredns.eu.org) is a privacy-focused DNS service that prioritizes user security and anonymity. We support both standard DNS and encrypted protocols while maintaining a strict no-logging policy.
```
Comment (el-termikael) [inline docs/general/dns-providers.md:358]:
```suggestion
[FutureDNS](https://futuredns.eu.org) is a privacy-focused DNS service that prioritizes user security and anonymity. It supports both standard DNS and encrypted protocols while maintaining a strict no-logging policy.
```

File: docs/general/dns-providers.md [hunk 2]
```diff
@@ -353,6 +353,25 @@ A collaborative open project to promote, implement, and deploy [DNS Privacy](htt
...
+| DNS-over-HTTPS, IPv4    | `https://dns.de.futuredns.eu.org/dns-query/`     | [Add to AdGuard](adguard:add_dns_server?address=https://dns.de.futuredns.eu.org/dns-query/&name=futuredns), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=https://dns.de.futuredns.eu.org/dns-query/&name=futuredns) |
+| DNS-over-HTTPS, IPv6    | `https://dns.de.futuredns.eu.org/dns-query/`     | [Add to AdGuard](adguard:add_dns_server?address=https://dns.de.futuredns.eu.org/dns-query/&name=futuredns), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=https://dns.de.futuredns.eu.org/dns-query/&name=futuredns) |
+| DNS-over-TLS            | `tls://dns.de.futuredns.eu.org`                  | [Add to AdGuard](adguard:add_dns_server?address=tls://dns.de.futuredns.eu.org&name=FutureDNSDoT), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=tls://dns.de.futuredns.eu.org&name=FutureDNSDoT) |
+| DNS-over-QUIC           | `quic://dns.de.futuredns.eu.org`                 | [Add to AdGuard](adguard:add_dns_server?address=quic://dns.de.futuredns.eu.org&name=FutureDNSDoQ), [Add to AdGuard VPN](adguardvpn:add_dns_server?address=quic://dns.de.futuredns.eu.org&name=FutureDNSDoQ) |
+
```
Comment (el-termikael) [inline docs/general/dns-providers.md:370]: please leave only one blank line

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Comment by futureshield** [comment]
i think it's fixed now sorry

**Review by el-termikael — APPROVED** [review]

**Comment by el-termikael** [comment]
@futureshield thank you for your contribution! now it's merged

**Pull request merged**